docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1612,6 +1612,109 @@ Note/limitations:
 
 - Applying a uniform non-response rate limits the impact of race/ethnicity, age, and sex to affect the sampled population. This might make some aspects of PRL easier as it is less likely the same simulants will be missing from each sample.
 
+Women, Infants, and Children (WIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+WIC is a government benefits program designed to support mothers and young 
+children. The main qualifications are income and presence of young children 
+in the home. 
+
+
+**When to Sample** 
+
+- Sample compiled on the time step containing Jan 1st of each year (the time step might end on Jan 2nd, Jan 15th, Jan 27th, etc.)
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Household number (random indicator which is the same for simulants in the same household)
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Home Address 
+  * - Home Zip Code 
+
+Here is an example: 
+
+.. image:: survey_example.png
+
+**Who to Sample** 
+
+Selection for the observer will be in 2 steps: 
+
+#. Eligibilty for WIC 
+#. Covered by WIC 
+
+Please note that multiple people in the same household can be selected and selection 
+should be independent. This applies to siblings (e.g., having one child in the household 
+on WIC does not mean a second child is or is not on WIC) or parent/child groups (a mother 
+being on WIC does not mean her infant is or is not on WIC). Inclusion is at the simulant 
+level. Do not include others in the household on the observer. 
+
+**Eligibility for WIC**
+
+Eligibility for WIC is based on income and age/children in the house. 
+To qualify you must be both: 
+
+- A child under the age of 5 (0-4 years old)
+- OR a tracked mother OR guardian of a child under the age of 1 
+
+And also: 
+
+- Have a total household income less than the limit below (based on number of people in the household)
+
+Max income = $16,410 + ($8,732 * number of people in the household) 
+
+For example, a one person household would have a maximum income of $25,142 and 
+a two person household would have a maximum income of $33,874. 
+
+Source: [WIC_eligibility]_
+
+**Covered by WIC**
+
+Once someone is found to be eligible, they are then randomly assigned to be 
+covered or not covered by WIC. This is a random sample based on the age of 
```

Comment:
I think we should either include both parent and child or neither --- it would be weird for the parent to get the benefit but not the child.

Reply:
Okay, I can see saying if the parent has benefits then the infant definitely does, especially since infant rates are higher than postpartum women. But the other direction (child has benefits --> mother must also) I don't think we should specify. The rates for mothers are lower and they requirements don't always match up (breastfeeding for moms; also having "nutritional risk" is a requirement that might be there for an infant but not a mother)

Reply:
I feel like you might need to spell out in more detail how you are introducing this dependency. You sort of hint at it with "... additional infants can be selected to reach the higher coverage rate for this group" but then in this section you are still saying it is a random sample at the coverage rates.

I think it is something like:
- Randomly select eligible tracked parents or guardians to enroll based on their coverage rate (84.7%)
- Also enroll _all_ (including multiple per parent/guardian) infants of these parents/guardians
- Calculate the infant coverage after the previous step
- Enroll more infants randomly, among those eligible but not already enrolled, _until_ the overall infant coverage is 98.4%.

For that last step, there is some math needed to figure out the coverage rate; if the engineers implement it as a rate, which I expect they will, we might want to do the math on our end just to make sure everyone is on the same page.

Reply:
Also @aflaxman it seems worth noting that this relationship between infant enrollment and parent enrollment shouldn't matter for our case study of counting under-5s, since the parent enrollment is not used in the case study (IIUC).

Reply:
> Also @aflaxman it seems worth noting that this relationship between infant enrollment and parent enrollment shouldn't matter for our case study of counting under-5s, since the parent enrollment is not used in the case study (IIUC).

Let's check in with @NathanielBlairStahn about that... I thought that the strategy would be to link the grownups and then see if the children were also included in the decennial --- any grownup/child pair in WIC that matches just the grownup in decennial is an opportunity to add a kid that was missed in the right place.

Reply:
@aflaxman Seems like we could also just look for all the kids receiving WIC, regardless of whether their parent does, and then try to match the child's parent from the WIC data (will it have info on children's parents even if the parents are not covered?) to the census data.

Reply:
Currently, no. The way its set up, it tracks only people receiving benefits. We could change this to always include guardians or always include the whole household. 

Both of these have their own issues: guardians in our current sim can die or move and not be replaced. And the household might include people not in the same family.

Reply:
@SylLutze If I'm understanding correctly, data on the child's parent will **not** be included in the WIC observer unless the parent is also on WIC? In that case does the observer record the fact that the two recipients have a parent-child relationship?

If we *don't* have parent information in the WIC data, then we can still try to link all the children to the census using other fields such as last name and address (and also full name and age/DOB for children who _are_ in the census data).

Reply:
That is correct. We do have a "household" indicator which says if 2 sims are in the same household - this could be parent/child or siblings. 

Again, we can change that to something else still if we need additional information.

Reply:
Ah, right, I hadn't paid attention to the household ID since I was just looking through this quickly. I think household number will be sufficient for the linking task when the child has a parent or other household member who is also covered by WIC. So I think the remaining question is whether the actual WIC data used by Census would have information about a covered child's parent(s) even if the parent is not covered; if so, that would make the linking task easier.

Reply:
Also, does WIC record other demographic data such as race/ethnicity that would also be captured by the decennial census? If so, that would also help with linking.